src/main/java/it/pagopa/transactions/client/EcommercePaymentMethodsClient.java

```diff
@@ -57,7 +58,12 @@ public Mono<PaymentMethodResponseDto> getPaymentMethod(
                                                            String paymentMethodId,
                                                            String xClientId
     ) {
-        return ecommercePaymentMethodsWebClientV1.getPaymentMethod(paymentMethodId, xClientId)
+        // payment methods only support CHECKOUT and IO.
```

Comment:
maybe here we can add a task to trace the CHECKOUT_CART and CHECKOUT methods discrimination since GEC allow bundle configuration for cart and single payment as independent configurations

Reply:
We decide not to distinguish `CHECKOUT_CART` from `CHECKOUT`, at least for now, right?